libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -354,23 +334,23 @@ void AP_Motors6DOF::output_armed_stabilizing_vectored()
     }
 
     // calculate roll, pitch and yaw for each motor
-    for (i=0; i<AP_MOTORS_MAX_NUM_MOTORS; i++) {
+    for (int8_t i = 0; i < AP_MOTORS_MAX_NUM_MOTORS; i++) {
         if (motor_enabled[i]) {
             rpy_out[i] = roll_thrust * _roll_factor[i] +
                          pitch_thrust * _pitch_factor[i] +
                          yaw_thrust * _yaw_factor[i];
         }
     }
 
-    float forward_coupling_limit = 1-_forwardVerticalCouplingFactor*float(fabsf(throttle_thrust));
+    float forward_coupling_limit = 1-_forwardVerticalCouplingFactor * fabsf(throttle_thrust);
```

Comment:
I do not know why whitespace is not added before and after the operator. I think that it is better to put it all if inserted.